README.md

```diff
@@ -39,14 +39,17 @@ test('Test the resolver', () => {
     },
   };
 
-  // parser.resolve() automatically typecasts (note JSON becomes a JS object for ease of testing)
   const response = parser.resolve(context);
 
-  // For convenience, the response is returned as a JS object rather than JSON
```

Comment:
It will only return a string if you have an error in your code (or you are not returning an object - which isn't supported with Appsync).